### DISCUSSION TITLE
Data loading for Concept Child with more than one Parent

### DISCUSSION THREAD

**Discussion by lawdank** [uwrit/leaf#595] [Q&A]
Hi Leaf Folks,

For a concept child with more than one parent we have noticed, the data loads for only one of them. Is this expected behavior or a bug? 

Eg : For **Anesthetics** under both _Lab Results and Measurements -> Drug Doses_ & _Lab Results and Measurements -> Drug Toxicology -> Drugs_

<img width="567" alt="image" src="https://github.com/uwrit/leaf/assets/26530107/bd2d1972-2100-4c3e-84b0-00e95ba6a38a">

<img width="470" alt="image" src="https://github.com/uwrit/leaf/assets/26530107/ec6f6150-cf57-45ef-87e0-acbb750a3f4b">

Thanks!

**Comment by ndobb** [comment]
Hi @lawdank, a given Leaf concept can have only one parent (this is [defined in the app.Concept table](https://github.com/uwrit/leaf/blob/master/src/db/obj/app.Concept.Table.sql#L15)). 

However as this looks like the LOINC tree, I assume the issue here is due to labs & lab categories within LOINC occasionally appearing multiple times in the tree (ie, having multiple parents). Is that correct?

If that's the case, then how concepts behave and where they appear under depend on the scripts used to populate the app.Concept table. In this example, I assume the script you've run outputs 2 `LP31466-3` concepts, each of which, within the Leaf app.Concept table, has a separate `Id`. Your script perhaps then chose just one of those `Id`s to use in populating the `ParentId` of Anesthetics child concepts. Thus the child concepts appear under only one of them.

Please let me know if this makes sense, or if I'm misunderstanding your question.
